### PR TITLE
Added github-site files

### PR DESCRIPTION
**Pull request by f1yn**

### CODE REVIEW

**Comment by stasm** [comment]
@flynnham Thanks for this :) I was busy with work these past few days. I'll review this tomorrow. I'm sure it's great though!

**Comment by f1yn** [comment]
@stasm Just noticed that the documentation header link was facing my fork and not the official branch. This has been patched.

**Comment by stasm** [comment]
@flynnham I see that you made this PR against the `gh-pages` branch. If I merge it, will `npm run deploy` continue to work? I use it to update the examples. If not, I'd be fine with not deploying the examples at all.

Thanks for the amazing work here. I like the structure of the page.  I think it's a bit too dark and too gray. I'd suggest being a bit bolder with your use of black and white :) I also have to admit that I don't understand what role the background animation serves. It's a cool canvas demo, still.

Last but not least, just taking the copy from the README might not cut it :) Perhaps we can work on a better copy in a follow-up.

**Comment by f1yn** [comment]
@stasm I'm not 100% sure about the `npm deploy`, since I've never deployed an npm package to `npm`. The theme was based on @GoldenRust's design. I think it's okay (definitely higher class for a GitHub page, but that's no means a bad thing), and yeah the background animation is just there because it's cool. As a neat feature, while the bg animation is disabled it still dynamically resizes with the page. :)

I have no experience with WebGL but I'm sure it would have been a better option than using canvas. It's disabled by default to help prevent potential lag on unoptimized canvas browsers and mobile.

In the future I'd like to do some custom writing for it as well. I haven't done anything on medium in a while, and my last draft was technically stolen by a publisher I write for... So this might be a good opportunity to do some writing. :)

If you need anything specific done with this commit, even if it's just changing the base branch let me know. I'll try to get to it when I can.

File: site/src/components/animationControl.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import html from '../../../index'; // import innerself
+import { connect } from '../reducers/store';
+
+function AnimationControl({ bgAnimation }){
+	let bgCanvas = window._bgCanvas;
+
+	console.log('bgCanvas', bgCanvas);
```
Comment (bcruddy) [inline site/src/components/animationControl.js:7]: Lets get rid of the `console.log`

**Comment by bcruddy** [comment]
Wow. This looks incredible. The router's simplicity and the "division of labor" between the components is perfect - nothing does too much, nothing does too little.

Only issue - are we sure about the `<canvas>` animations? I'm on a machine with 16 GB of RAM and even minutes after the page loaded it was a bit jumpy. 

I don't want the site to over-sell `innerself` because it really is brilliant for building quick apps for yourself or PoCs. For me, `innerself` is there to be a simple thing to turn to when I want to build something quick and abstract state management. Nothing fancy - just do the job and when I need to bring it to a production environment its getting rewritten in angular/vue/react.

**Comment by f1yn** [comment]
@bcruddy Canvas performance seems to dip on specific browsers [namely Firefox and some versions of IE]. I actually use a derivative of Firefox as my daily, so I might actually just scrap `canvas-bokeh`. It was mostly conceptual to begin with, and as I mentioned I might have to come back to it someday with a more suited technology (namely WebGL).

**Comment by f1yn** [comment]
I've decided to leave the canvas-bokeh, but to disable the animations and prevent the user from having the option to enable them. It still preforms rather well for a bg-rendering. I've been super busy lately, so it's nice to finally get a break to address these things.

File: site/src/components/animationControl.js (hunk 1 above)
Comment (f1yn) [reply site/src/components/animationControl.js:7]: I'm more than willing to remove this one, but the `hey!` is there to stay.

**Comment by f1yn** [comment]
@stasm, in regards to the site, I think it might also be possible to upload the site to **master** under the **/docs** option. I will likely be experimenting with github-pages and npm publications the next couple of days to see how it all would work.

**Comment by f1yn** [comment]
@stasm Just wondering if you are still alive. We all get busy, so no worries there. If you're available I'd love some pointers about possibly writing new content for the site.

From my last pushes I've updated the UI so it's overall less dark looking. If you have any suggestions for updates just throw them my way. I've been using innerself for some of my own projects so I'd be glad to contribute back :smile:

**Pull request closed without merging**